common/app/views/fragments/contentFooter.scala.html

```diff
@@ -19,7 +20,7 @@
     } {
         <div class="js-fronts-network-2" data-link-name="fronts-on-articles | network-front | block-2"></div>
     } {
-        @if(content.commercial.needsHighMerchandisingSlot) {
+        @if(content.commercial.needsHighMerchandisingSlot(Edition(request).displayName)) {
```

Comment:
Better to pass through whole `Edition` instance so that you get the benefit of knowing the type in the method signature